src/HigherOrderComponent/MappifiedComponent/MappifiedComponent.spec.jsx

```diff
@@ -51,12 +49,14 @@ describe('mappify', () => {
     });
 
     it('warns if no map is given in the context', () => {
-      const loggerSpy = sinon.spy(Logger, 'warn');
+      const loggerSpy = jest.spyOn(Logger, 'warn');
       TestUtil.mountComponent(EnhancedComponent);
-      expect(loggerSpy.calledOnce).toBeTruthy();
-      expect(loggerSpy.calledWith('You trying to mappify a component without any map in the '+
-      'context. Did you implement the MapProvider?')).toBeTruthy();
-      loggerSpy.restore();
+      expect(loggerSpy).toHaveBeenCalled();
+      expect(loggerSpy.mock.calls[0][0]).toBe('You trying to mappify a ' +
+        'component without any map in the context. Did you implement ' +
+        'the MapProvider?');
```

Comment:
We might better use `.toHaveBeenCalledWith`  https://facebook.github.io/jest/docs/en/expect.html#tohavebeencalledwith